packages/api/source/controllers/MembershipController.ts

```diff
@@ -0,0 +1,155 @@
+import type { Document } from "mongoose";
+import joi from "joi";
+import {
+    MembershipModel,
+    UserModel,
+    constants,
+    BadRequestError,
+    NotFoundError,
+    OrganizationModel,
+    ExternalMembership,
+    runAsTransaction,
+    sendEmail,
```

Comment:
Why do you need `@sendgrid/mail` when you have `sendEmail`?

Reply:
Also, `sendEmail` should be in `@hypertool/api`, not `@hypertool/common`. IIRC, I told you to create a `utils` module.